bentoctl_lambda/aws_lambda/bentoctl_user_template.j2

```diff
@@ -3,7 +3,10 @@
 {% block SETUP_BENTO_ENTRYPOINT %}
 EXPOSE 3000
 
-RUN --mount=type=cache,mode=0777,target=/root/.cache/pip        pip install awslambdaric==2.0.0 mangum==0.12.3
+RUN --mount=type=cache,mode=0777,target=/root/.cache/pip \
```

Comment:
how would we start a bento again? since you are overwritting ENTRYPOINT

Reply:
the CMD command is configured my the terraform template but I'll put it back here since that is less confusing